cypress/integration/nav_bar.ts

```diff
@@ -50,4 +50,18 @@ describe("Nav Bar", () => {
       .should("have.attr", "href")
       .and("include", LEGACY_URLS.distros);
   });
+  it("Nav Dropdown should link to patches page of most recent project if cookie exists", () => {
+    cy.setCookie("mci-project-cookie", "spruce");
+    cy.visit(SPRUCE_URLS.userPatches);
+    cy.dataCy("auxiliary-dropdown-link").first().click();
```

Comment:
nit. is `.first()` necessary are there multiple elements with this data-cy selector?